doc/getting_started/getting_started.rst

```diff
@@ -0,0 +1,73 @@
+Getting Started
+===============
+
+Install ROS and Catkin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+`Install ROS Kinetic <http://wiki.ros.org/kinetic/Installation/Ubuntu>`_.
+It is easy to miss steps when going through the ROS installation tutorial. If you run into errors in the next few steps, a good place to start is to go back and make sure you have installed ROS correctly.
+
+Once you have ROS installed, make sure you have the most up to date packages: ::
+
+  rosdep update
+  sudo apt-get update
+  sudo apt-get dist-upgrade
+
+Install `catkin <http://wiki.ros.org/catkin>`_ the ROS build system: ::
+
+  sudo apt-get install ros-kinetic-catkin python-catkin-tools
+
+Install MoveIt!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The simplest way to install MoveIt! is from pre-built binaries (Debian): ::
+
+  sudo apt install ros-kinetic-moveit
+
+Advanced users might want to `install MoveIt! from source <http://moveit.ros.org/install/source/>`_.
+
+Create A Catkin Workspace
+^^^^^^^^^^^^^^^^^^^^^^^^^
+You will need to have a `catkin <http://wiki.ros.org/catkin>`_ workspace setup: ::
+
+  mkdir -p ~/ws_moveit/src
+
+Download the Example Code
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Within your `catkin <http://wiki.ros.org/catkin>`_ workspace, download these tutorials: ::
+
+  cd ~/ws_moveit/src
+  git clone https://github.com/PickNikRobotics/moveit_tutorials.git
+
+You will also need a ``panda_moveit_config`` package to follow along with these tutorials:
+
+  git clone https://github.com/PickNikRobotics/panda_moveit_config.git
```

Comment:
change this

move the repo to ros-planning, lmk if you need permission for this

Reply:
I have just moved it:

https://github.com/ros-planning/panda_moveit_config